### PR TITLE
Add missing test for image reloading bug

### PR DESCRIPTION
**Pull request by jdm**
This adds an automated test for #22336.

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/22356)

### CODE REVIEW

File: tests/wpt/web-platform-tests/html/semantics/embedded-content/the-img-element/available-images-onload.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<html>
+<title>Ensure images from available images list can be drawn to a canvas</title>
```
Comment (Manishearth) [inline tests/wpt/web-platform-tests/html/semantics/embedded-content/the-img-element/available-images-onload.html]: spec link about available images list?

**Review by Manishearth — APPROVED** [review]

**Comment by jdm** [comment]
@bors-servo retry
* cleaned up linux-dev build directory on servo-linux4

**Comment by jdm** [comment]
@bors-servo retry
* #22062

**Pull request merged**